about.html

```diff
@@ -24,7 +24,34 @@
     </style>
   </head>
   <body>
+    <div class="mobile-dropdown hidden">
+      <div class="close-x">
+        <img src="./assets/img/close-x.png" alt="close dropdown" width="80" />
+      </div>
+      <div class="menu-list">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Best Seller</a></li>
+          <li><button type="button">Presentation</button></li>
+        </ul>
+      </div>
+    </div>
     <header class="header">
+      <div class="dark-header bg-secondary">
+        <ul class="d-flex text-white p-2 justify-content-end">
+          <li class="me-3">
+            <img src="./assets/img/facebook.png" alt="Facebook logo" />
+          </li>
+          <li class="me-3">
+            <img src="./assets/img/twitter.png" alt="Twitter logo" />
+          </li>
+          <li class="me-3">English</li>
+          <li class="me-3">My page</li>
+          <li class="me-3">Logout</li>
+        </ul>
```

Comment:
- [ ] good job adding the top-bar kindly use the correct color wich is black to match the template design 

![image](https://user-images.githubusercontent.com/89282221/190503709-fa5d5162-a129-4442-a529-c75893b5c7bd.png)